### ISSUE TITLE
Many payloads don't work due to not using DuckyScript V1

### ISSUE THREAD

**Issue by zPushz** [aleff-github/my-flipper-shits#15]
Didn't look through all of them but Change_the_password_of_the_windows_user uses "DEFINE" and "left-arrow" when define isn't supported at all and it should be "leftarrow"

Change_windows_user_name also uses "DEFINE"

ExfilitrateComputerScreenshots also uses "DEFINE"

**Comment by aleff-github** [comment]
Yes I confirm, I included this item in the Faq section some time ago.

https://github.com/aleff-github/my-flipper-shits?tab=readme-ov-file#faqs
